debian/changelog

```diff
@@ -1,3 +1,11 @@
+libsilkit (4.0.53-1ubuntu1) UNRELEASED; urgency=medium
+
+  [ Andreas Rentschler ]
```

Comment:
do we really need to duplicate our changelog here? isn't this a meta changelog for the *debian packages*, e.g. wouldn't the changelog entry be more like "revision x+1: switch to python based build script" ?

Reply:
So my GOTO role model is the curl package, and after reading their changelog you might be correct. I think it is fine to record upstream changes. But we do not need to be so verbose and mention the upstream contributors. Unless we want to give them the exposure because of all the people that read DEBIAN changelogs :)

I would not change this for history keeping, but maybe for >= 4.0.53-2 we can use a more terse changelog 👍